projects/1.18-fabric/assets/columns/columns/lang/zh_cn.json

```
@@ -0,0 +1,23 @@
+{
+	"block.columns.andesite_column": "安山岩立柱",
+	"block.columns.blackstone_column": "黑石立柱",
+	"block.columns.brick_column": "红砖立柱",
```

```suggestion
	"block.columns.brick_column": "砖立柱",
```
红砖的改动是在1.19作出的，1.18及以下使用`砖`

---

已改